src/WinCache.php

```diff
@@ -75,9 +75,9 @@ public function has($key): bool
     /**
      * @noinspection PhpDocMissingThrowsInspection DateTime won't throw exception because constant string is passed as time
      *
-     * Normalizes cache TTL handling `null` value and {@see DateInterval} objects.
-     * @param int|DateInterval|null $ttl raw TTL.
-     * @return int|null TTL value as UNIX timestamp or null meaning infinity
+     * Normalizes cache TTL handling `null` value, strings and {@see DateInterval} objects.
+     * @param int|string|DateInterval|null $ttl raw TTL.
```

Comment:
Why do we want strings there?

Reply:
Although PSR interface has doc blocks `@param null|int|\DateInterval $ttl` for the methods, but it cannot be type hinted. So, a string can be also passed.
Alternatively, we can throw `InvalidArgumentException` if a string is passed.

Reply:
Casting should be fine.